content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Can you rename content/1.7/back-office/shop-parameters/search/tags/add-edit-tags.md by content/1.7/back-office/shop-parameters/search/tags/add-edit-tag.md as you add and edit only one tag, the S should be removed